thirdeye/thirdeye-frontend/.gitignore

```diff
@@ -7,6 +7,7 @@
 # dependencies
 /node/
 /node_modules
+/bower_components
```

Comment:
Is there any way we can keep bower out of the frontend?

Reply:
Yes, we should be. This just auto-updated with `ember-cli-update`. Since we do not have a `bower.json`, the bower_components folder will not be updated.

Reply:
I have removed those entries now from this file to remove any confusion.